content/en/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -19,7 +19,7 @@ This document catalogs the communication paths between the control plane (really
 {{% capture body %}}
 
 ## Node to Control Plane
-All communication paths from the nodes to the control plane terminate at the apiserver (none of the other master components are designed to expose remote services). In a typical deployment, the apiserver is configured to listen for remote connections on a secure HTTPS port (443) with one or more forms of client [authentication](/docs/reference/access-authn-authz/authentication/) enabled.
+All communication paths from the nodes to the control plane terminate at the apiserver (none of the other master components are designed to expose remote services). In a typical deployment, the apiserver is configured to listen for remote connections on a secure HTTPS port (6443) with one or more forms of client [authentication](/docs/reference/access-authn-authz/authentication/) enabled.
```

Comment:
I'm not sure that 6443 is less typical that 443.
(I'm an IT consultant and I've seen a mix of deployments, I think more with the API server reachable on port 443 than on any other port).

That said, maybe it's worth adding a `{{< note >}}` or an aside to mention that the default port is not 6443, or maybe to reword to hint that the port you _reach_ the API server on might not be the same as the port that the process thinks you're connecting to it on.

Reply:
@sftim thanks for the review. Just to confirm 443 you mentioned in deployments is the kube-apiserver secure port, not the kubernetes service port? Asking because I know clients do reach Kubernetes API on port 443, but that's the port of the "kubernetes" service and will be DNATed to 6443 finally.
I thought this was out of date and saw another document using 6443 https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/#transport-security so made the change. If this is by intention, I will abandon this PR.

Reply:
When the node talks to the control plane, I think that's most commonly on a TCP connection where the client (kubelet) thinks it is connecting to port 443.

Reply:
Agree, but the document here is "the apiserver is configured to listen for remote connections on a secure port", I think it should be which the server listens on.

Reply:
The _heading_ here is Node to Control Plane; bear that in mind too.

Reply:
I understand the context of this description but don't understand how it affects the port's number which indicates the port the apiserver is listening on.. If it's describing which port a Node will connect to or something ambiguous, I would agree with you. 
BTW, even from Node's perspective, 6443 is also widely used as the destination port for kubelet, for example in some deployment tools like kubeadm and kind, if it's a single control-plane node deployment, kubelet usually access master:6443 directly (both for kubeadm and kind), if it's a multiple control-plane nodes deployment, there will be a loadbalancer listening on port 6443 by default (kind does this, kubeadm doesn't set loadbalancer automatically and up to user to set the control-plane-endpoint)

Reply:
Thanks @neolit123 and @sftim for your input. Since the words here is about the HTTPS port kube-apiserver typically listens on, I think it should be 6443 to be consistent with the code and most documents. Please let me know if you think this should be kept as it was.

Reply:
I'd like an _and_ kind of approach: mention port 443 (as per web standards etc) _and_ mention port 6443 (as hard coded into `kube-apiserver`, used in single host kubeadm deployments, etc).